core/src/main/java/io/questdb/cutlass/pgwire/modern/PGPipelineEntry.java

```diff
@@ -1391,6 +1391,10 @@ private void msgExecuteSelect(
 
             sqlExecutionContext.getCircuitBreaker().resetTimer();
             sqlExecutionContext.setCacheHit(cacheHit);
+            if (preparedStatement && !cacheHit) {
```

Comment:
nit: could be simpilified to
```suggestion
            if (preparedStatement) {
```